### DISCUSSION TITLE
Size and Fit question

### DISCUSSION THREAD

**Discussion by akhepcat** [rowokii/A2C_TangyPack#4] [General]
This looks like a great option, and I'd really like to get one!

My question is simple:  Does the pack have enough clearance on the left side so that it can be mounted side-by-side with a FujiNet adapter?

thanks!

**Comment by rowokii** [comment]
I don't have a FujiNet adapter, so I can't comment on if it will fit side by side, but this is what it looks like attached to the back of Apple IIc+ with a disk drive connected. 

![PXL_20240912_013607965](https://github.com/user-attachments/assets/a6a2c316-1211-4b6a-8eb0-86ec74572686)

**Reply by akhepcat** [reply]
Yeah, the HDMI port wing and cable definitely blocks a FujiNet.   Sigh.
